src/browser/modules/D3Visualization/components/StyleableRelType.tsx

```diff
@@ -24,16 +24,16 @@ import { Popup } from 'semantic-ui-react'
 import { GraphStyle } from './OverviewPane'
 import { StyledRelationship } from 'browser/modules/DBMSInfo/styled'
 
-export type StylableRelTypeProps = {
+export type StyleableRelTypeProps = {
```

Comment:
Hehe nice spelling fix. I actually had to google this to check which one was correct :-D

Reply:
hehe it was my IntelliJ who complained :)